src/content/docs/en/guides/integrations-guide/netlify.mdx

```diff
@@ -157,7 +157,7 @@ To opt out of Netlify's Image CDN remote image optimization, use the `imageCDN`
 
 ```js title="astro.config.mjs" ins={8}
 import { defineConfig } from 'astro/config';
-import netlify from '@astrojs/netlify/functions';
+import netlify from '@astrojs/netlify';
```

Comment:
@ascorbic why the update of the import?

Reply:
The code sample was wrong. The import changed in a previous version.

Reply:
See https://github.com/withastro/adapters/blob/main/packages/netlify/src/functions.ts